### PR TITLE
add description about PessimisticLockNotFound relating to lazy uniqueness check

### PR DESCRIPTION
**Pull request by ekexium**
Signed-off-by: ekexium <eke@fastmail.com>

### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.4 (TiDB 6.4 versions)
- [x] v6.3 (TiDB 6.3 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: constraints.md [hunk 1]
```diff
@@ -246,6 +246,8 @@ ERROR 1062 (23000): Duplicate entry 'bill' for key 'username'
     ERROR 9007 (HY000): Write conflict, txnStartTS=435688780611190794, conflictStartTS=435688783311536129, conflictCommitTS=435688783311536130, key={tableID=74, indexID=1, indexValues={bill, }} primary={tableID=74, indexID=1, indexValues={bill, }}, reason=LazyUniquenessCheck [try again later]
     ```
 
+- 关闭该变量时，如果多个悲观事务之间有写冲突，悲观锁可能会被其它悲观事务提交时强制回滚，因此产生 `Pessimistic lock not found` 错误。发生该错误时，说明该业务不适合推迟悲观事务的唯一约束检查，应考虑业务上避免冲突，或在错误后重试事务。
```
Comment (qiancai) [inline constraints.md:249]:
```suggestion
- 关闭该变量时，如果多个悲观事务之间存在写冲突，悲观锁可能会在其它悲观事务提交时被强制回滚，因此产生 `Pessimistic lock not found` 错误。发生该错误时，说明该业务不适合推迟悲观事务的唯一约束检查，应考虑调整业务避免冲突，或在发生错误后重试事务。
```

**Review by TomShawn — APPROVED** [review]

**Comment by qiancai** [comment]
Removed the **needs-cherry-pick-release-6.3** label because the v6.3 docs have been archived at https://docs-archive.pingcap.com/zh/tidb/v6.3 and will no longer receive new updates.

**Review by qiancai — APPROVED** [review]
LGTM

File: constraints.md [hunk 2]
```diff
@@ -246,6 +246,8 @@ ERROR 1062 (23000): Duplicate entry 'bill' for key 'username'
     ERROR 9007 (HY000): Write conflict, txnStartTS=435688780611190794, conflictStartTS=435688783311536129, conflictCommitTS=435688783311536130, key={tableID=74, indexID=1, indexValues={bill, }} primary={tableID=74, indexID=1, indexValues={bill, }}, reason=LazyUniquenessCheck [try again later]
     ```
 
+- 关闭该变量时，如果多个悲观事务之间存在写冲突，悲观锁可能会在其它悲观事务提交时被强制回滚，因此产生 `Pessimistic lock not found` 错误。发生该错误时，说明该业务不适合推迟悲观事务的唯一约束检查，应考虑调整业务避免冲突，或在发生错误后重试事务。
```
Comment (MyonKeminta) [inline constraints.md:249]: Should the error message be `PessimisticLockNotFound`?

**Review by MyonKeminta — COMMENTED** [review]
Rest LGTM

**Comment by qiancai** [comment]
Removed the needs-cherry-pick-release-6.4 label because the v6.4 docs have been archived at https://docs-archive.pingcap.com/zh/tidb/v6.4 and will no longer receive new updates.

**Pull request merged**